ibis/expr/format.py

```diff
@@ -447,12 +447,18 @@ def type_info(datatype: dt.DataType) -> str:
 def _fmt_selection_column_value_expr(
     expr: ir.Value, *, aliases: Aliases, maxlen: int = 0
 ) -> str:
-    raw_name = expr._safe_name
-    assert raw_name is not None, (
-        "`_safe_name` property should never be None when formatting a "
-        "selection column expression"
-    )
-    name = f"{raw_name}:"
+    if isinstance(expr, ir.DestructValue):
```

Comment:
Since this function is dispatched, can you handle `DestructValue` in a separate function? `DestructValue` already doesn't conform to ibis's architecture and I'd like to avoid its special cases in generic code.

Reply:
Oh good point—done